tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -231,7 +231,7 @@ public void init() {
 	 * 
 	 * @return the gradebook for the site
 	 */
-	private Gradebook getGradebook() {
+	public Gradebook getGradebook() {
```

Comment:
Yeah, alternatively a new public API method to get that info, keeping the Gradebook private.